### PR TITLE
IDEM-25

### PR DESCRIPTION
**Pull request by brettvniekerk**
- [Link to ticket](https://github.com/dltxio/idem-site/issues/25)
- Added new QR code and responsive modal for mobile

Desktop:
image

Mobile:
image

### CODE REVIEW

File: src/components/QrCodeModal.tsx [hunk 1]
```diff
@@ -10,25 +11,48 @@ type Props = {
 };
 
 const QrCodeModal: React.FC<Props> = (props) => {
+  const size = useWindowSize();
   const { isModalOpen, setIsModalOpen, buttonTitle, buttonColor } = props;
 
+  const testflight = process.env.VITE_TESTFLIGHT_URL;
```
Comment (xiangzhe0812) [inline src/components/QrCodeModal.tsx:17]: Not TestFlight anymore

Comment (brettvniekerk) [reply src/components/QrCodeModal.tsx]: done

File: src/components/QrCodeModal.tsx [hunk 2]
```diff
@@ -10,25 +11,48 @@ type Props = {
 };
 
 const QrCodeModal: React.FC<Props> = (props) => {
+  const size = useWindowSize();
   const { isModalOpen, setIsModalOpen, buttonTitle, buttonColor } = props;
 
+  const testflight = process.env.VITE_APPSTORE_URL;
```
Comment (xiangzhe0812) [inline src/components/QrCodeModal.tsx:17]: Change the const name as well?

Comment (brettvniekerk) [reply src/components/QrCodeModal.tsx]: done

**Review by xiangzhe0812 — APPROVED** [review]

**Comment by bitcoinbrisbane** [comment]
@brettvniekerk neesds a screen shot

**Review by bitcoinbrisbane — APPROVED** [review]

**Review by xiangzhe0812 — APPROVED** [review]

**Pull request merged**